packages/main/src/ColorPalettePopover.ts

```diff
@@ -187,10 +187,25 @@ class ColorPalettePopover extends UI5Element {
 
 		this.responsivePopover!.showAt(opener, true);
 
+		const colorPalette = this._colorPalette();
+		const selectedItem = this.colors.find(item => item.selected)
+						  || colorPalette.recentColorsElements.find(item => item.selected);
```

Comment:
Are we going with this behaviour? That's the one that we were asked to to do at the start, but wasn't the lates that we have to focus the place it was selected from?

Reply:
As long as I remember the latest decision of our discussion was, that we are going with the behavior, which is when a Popover is opened, the `selected` item if such exists should be focused ? Am I wrong ?

Reply:
You're right, I missed that while testing other scenario behaviours, thanks. Im fixing it